downscale/estimate_forcing_regression.f90

```diff
@@ -440,10 +486,29 @@ end subroutine heapsort
   print *, 'Elapsed time for weight generation: ', real (t2-t1) / real (count_rate)
 
   ! AWW-Feb2016:  just allocate grids once time, and re-use in code below
-  allocate (twx_red(6, sta_limit))    ! these have dim1 = 6
-  allocate (tx_red(6, sta_limit))
-  allocate (twx_red_2(4, sta_limit))  ! these are for no slope calcs, have dim1 = 4
-  allocate (tx_red_2(4, sta_limit))
+  allocate (twx_red(nPredict, sta_limit))    ! these have dim1 = 6
+  allocate (tx_red(nPredict, sta_limit))
+  allocate (twx_red_2(nPredict-2, sta_limit))  ! these are for no slope calcs, have dim1 = 4
+  allocate (tx_red_2(nPredict-2, sta_limit))
+
+
+  !open NWP predcitor file list
+  open(unit=55,file=trim(nwp_input_list),form='formatted',status='old',iostat=error)
```

Comment:
we should add a check on the number of grids listed to make sure they are >= ntimes.  check for # of lines then rewind file.